src/rest_framework_jwt/authentication.py

```diff
@@ -89,11 +89,11 @@ def authenticate(self, request):
 
     @classmethod
     def get_token_from_request(cls, request):
-        authorization_header = force_str(get_authorization_header(request))
-
         try:
+            authorization_header = force_str(get_authorization_header(request))
```

Comment:
IMO `rest_framework.authentication.get_authorization_header` *could* check the header can be decoded - im not sure if it knows 100% how to decode the headers, but my guess is that it should be able to.